resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/config.scala

```diff
@@ -225,6 +225,18 @@ package object config extends Logging {
     .timeConf(TimeUnit.MILLISECONDS)
     .createWithDefaultString("1s")
 
+  private[spark] val REPORT_LOG_FREQUENCY = {
+    ConfigBuilder("spark.yarn.report.logging.frequency")
+      .doc("Maximum number of application reports processed " +
+        "until the next application status is logged. " +
+        "If there is a change of state, the application status will be logged " +
+        "regardless of the number of application reports processed. " +
+        "This property is dependent on the spark.yarn.report.interval")
```

Comment:
this property isn't dependent upon it, the overall frequency is dependent upon it.  this * frequency = time when logged when no status change.

Reply:
there is also a sark.yarn.clientLaunchMonitorInterval that is used, that is why the interval is passed into that function.